api.js

```diff
@@ -165,9 +155,25 @@ Api.prototype._prefixTitle = function (file) {
 Api.prototype.run = function (files) {
 	var self = this;
 
+	if (!files || files.length === 0) {
+		this.files = [
```

Comment:
Not needed since `cli.js` takes care of setting defaults. The watcher also receives those defaults, which is what it should provide when calling `run()`.